community/community.html

```diff
@@ -4,8 +4,43 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./community.css" />
-    <title>Document</title>
+    <title>Food App</title>
+    <link rel="stylesheet" href="community.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lobster&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body></body>
```

Comment:
Is this extra body attribute?!!

Reply:
No There is no problem, it only appeared when rearranging the codes